src/solver/optimisation/hourly_csr_problem.h

```diff
@@ -45,6 +45,7 @@ class HourlyCSRProblem
     int hourInWeekTriggeredCsr;
     double belowThisThresholdSetToZero;
     PROBLEME_HEBDO* problemeHebdo_;
+    PROBLEME_ANTARES_A_RESOUDRE problemeAResoudre_;
```

Comment:
Yeah we need it for many things:
- Adequacy patch parameters
- Adequacy patch runtime data (e.g areas inside/outside/virtual)
- Variable/constraint correspondence arrays necessary for
  - Accessing results from the adequacy simulation
  - Accessing the hourly binding constraint indices (input data)

It would be nice to remove it though.